config/application.rb

```diff
@@ -22,5 +22,6 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.secret_key_base = Figaro.env.av_dev_key_base
```

Comment:
convention is that global environment vars should be uppercase. i also prefer `ENV` to `Figaro.env`. so this should be:

    ENV["AV_DEV_KEY_BASE"]

or at least:

    Figaro.env.AV_DEV_KEY_BASE